hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson

```diff
@@ -250,6 +250,11 @@
       uvm_test_seq: rv_dm_jtag_dtm_hard_reset_vseq
       reseed: 2
     }
+    {
+      name: rv_dm_abstractcmd_status
+      uvm_test_seq: rv_dm_abstractcmd_status_vseq
+      reseed: 10
```

Comment:
Why 10 here? I can't see much being randomised in this vseq! Surely 1 or 2 would give the same confidence?